composables/party/useCursorDropEvents.ts

```diff
@@ -1,15 +1,16 @@
 import useParty from '@/composables/party/useParty'
 import { DropEventType } from './types'
-import { DropMintedNft } from '../drop/useGenerativeDropMint'
+import useGenerativeDropMint from '../drop/useGenerativeDropMint'
+import { useDrop } from '@/components/drops/useDrops'
 
 type EventParams = { image?: string; completed?: boolean }
 
-export default (
-  drop: string,
-  mintingWatch?: Ref<boolean>[],
-  mintedNft?: Ref<DropMintedNft | undefined>,
-) => {
-  const { sendMessage } = useParty({ room: computed(() => drop) })
+export default (mintingWatch?: Ref<boolean>[]) => {
+  const drop = useDrop()
+  const { mintedNft } = useGenerativeDropMint()
```

Comment:
should be like this?
```suggestion
  const { drop } = useDrop()
```

Reply:
it should.
thank you 


done